pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -368,6 +371,13 @@ var storageClassToProvisionerKeyMapper = map[string]func(sc *storagev1.StorageCl
 		}
 		return "ebs.csi.aws.com"
 	},
+	"pd.csi.storage.gke.io": func(sc *storagev1.StorageClass) string {
+		val := sc.Parameters["type"]
+		if val == "hyperdisk-balanced" {
```

Comment:
Got bitten before by a similar parameter, are you sure exact match is a must here? Maybe something more forgiving fits better like strings.Contains or?

Reply:
For instance if `hyperdisk-nonbalanced` is another option, you would fail here and fall back to non RWX.

Reply:
Something like this is what worried me:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/d7c5a23550f4fb4d44ec6f38cf7abf69492b6898/pkg/common/utils.go#L736
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/d7c5a23550f4fb4d44ec6f38cf7abf69492b6898/pkg/common/parameters.go#L80

Reply:
Per discussion with GCP folks and according to their [ref](https://cloud.google.com/compute/docs/disks/modify-hyperdisks#limits), I prefer adding RWX explicitly to the specific types. We will sure extend the coverage in the future if needed.